src/daemon/DaemonManager.cpp

```diff
@@ -59,6 +59,10 @@ bool DaemonManager::start(const QString &flags, NetworkType::Type nettype, const
     // prepare command line arguments and pass to monerod
     QStringList arguments;
 
+    arguments << "--zmq-pub" << "tcp://127.0.0.1:18083";
+    arguments << "--disable-dns-checkpoints";
+    arguments << "--enable-dns-blocklist";
```

Comment:
I would prefer if we not set this one by default.

Reply:
Fixed, would you rather have the arguments be reverted when stopping mining or leave it?